src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -170,9 +207,23 @@ public void saveCondition() {
 	@Then("the system adds New Condition in Conditions table")
 	public void systemAddsCondition() {
 		assertNotNull(conditionsPage.getConditionsList());
```

Comment:
Am expecting something like save, click Button since the scenario is referring to the system to add new condition , however the methods defined are meaning something different. Do you mind sharing more insights here

Reply:
Thanks Have rectified the method

Reply:
> Am expecting something like save, click Button since the scenario is referring to the system to add new condition , however the methods defined are meaning something different. Do you mind sharing more insights here

@sherrif10  that is already accomplished by this method
```
@And("a user clicks on save condition button")
	public void saveCondition() {
		conditionPage.clickSave();
	}
```
then since we deleted one condition and remained with one, thye following method is to assure us that the conditions table is not empty. It has some one condition saved

```
@Then("the system adds New Condition in Conditions table")
	public void systemAddsCondition() {
		assertNotNull(conditionsPage.getConditionsList());
```